usersFeed.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+sbt "project noah" "run-main edu.uci.ics.cloudberry.noah.feed.TwitterFeedUsersTimelineDriver \
+-ck Your Consumer Key \
+-cs Your secret consumer key \
+-tk Your token \
+-ts Your secret token \
+-tu "eZikaVirus_com"
+-u 127.0.0.1 -p 10001 -w 0 -b 50 -fo"
```

Comment:
-u -p -w -b are currently required options in the Config. If it is ok to change it, I can do it.